src/components/Stack.vue

```diff
@@ -297,7 +302,7 @@ export default {
             console.log('current active card: ' + this.$store.getters.getActiveCard)
           if (thisStack.cards.length === 0) {
 
-                $('button[stackId="'+this.stackId+'"]').attr("data-content", "You cannot add a variable card to a stack without an instruction card. Try adding that card to a stack with an instruction card." );
+                $('button[stackId="'+this.stackId+'"]').attr("data-content", "You can only add variable cards to a stack with an open variable (Rx) repetition card or an existing variable card." );
```

Comment:
I believe so, I was able to replace a v2 with a v4. Let me know if you want to keep that feature or not.